modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -224,138 +223,130 @@ xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database in cas
+ . Disable them
+ . Perform the updating procedure to the desired version
 
-The disabled processes with 6.x forms will not be able to be enabled again post migration.
-Having 6.x case overview pages on your processes will not prevent the migration of the platform,
-however they will all be replaced by the default 7.x case overview page, created with the UI Designer.
-It means that you might want to redo the case overview page as well as the forms, especially if you have configure
-a custom case overview page for your processes in version 6.x. Or (for Enterprise, Performance, and Efficiency editions only),
-you can live update it after migration.
+The disabled processes with 6.x forms cannot be enabled again post update. +
 
-[NOTE]
-====
+==== Case overview pages
 
-Note: 6.x application resources have been removed too in 7.8.0, so if you are migrating a process that leverage this feature, you need to modify it (for example to use process dependencies instead (Configure > Process dependencies in Bonita Studio)).
-====
+Having 6.x *case overview* pages on your processes will not prevent the update of the runtime. However, they will all be replaced by the default 7.x case overview page, created with the UI Designer.
+It means that you might want to redo the case overview pages as well as the forms, especially if you have configured a custom case overview page for your processes in version 6.x. +
+Alternatively, you can also xref:live-update.adoc[live update] the case overview page after the update is complete. (Enterprise, Performance, and Efficiency editions only).
 
 [#update-case-overview-pages]
 
-== Use the provided Bonita tool to update case overview pages before migrating to 7.8.0
+*Automated replacement of case overview pages during the updating procedure*
 
-Bonita Migration Tool now ships an option to allow you to replace 6.x case overview pages with the default 7.x case overview page
-(created with the UI Designer), when your Bonita runtime is still in a pre-7.8.0 version. This allows you to see if the page suits your needs, or if not,
-it can be used as a base to customize your case overview page. Your pages will then be ready for the 7.8.0 migration step.
+If the source version is older than Bonita 7.8. +
+Starting with Bonita 7.8, Bonita Migration Tool ships an option to allow you to replace 6.x case overview pages with the default 7.x case overview page (created with the UI Designer). This allows you to see if the page suits your needs. +
+If not, it can be used as a base to customize your case overview page. Your pages will then be ready for the 7.8.0 step of the Bonita Runtime update.
 
-To run it, unzip the latest Migration Tool and run, for *Community* edition: +
-`./bonita-migration-distrib` (Linux) or `bonita-migration-distrib.bat` (Windows) `--updateCaseOverview <PROCESS_DEFINITION_ID>`
+To run this option, unzip the migration tool and run:
```

Comment:
unzip the migration tool and run: => unzip the migration tool and execute commands as suggested below:

Reply:
fixed